src/mutations/system/deleteBlockedSearchKeyword.ts

```diff
@@ -0,0 +1,26 @@
+import { UserInputError } from 'common/errors'
```

Comment:
Since `deleteBlockedSearchKeyword` doesn't in schema, shall we delete this file?